paraview/xmls/MergeTreeAutoencoder.xml

```diff
@@ -0,0 +1,653 @@
+      default_values="100"
+      panel_visibility="advanced">
+        <Documentation>
+          Choose the number of iterations between two displays of information in the console.
+        </Documentation>
+      </IntVectorProperty>
+      
+       <DoubleVectorProperty
+      name="BatchSize"
+      command="SetBatchSize"
+      label="Batch Size"
+      number_of_elements="1"
+      default_values="1"
+      panel_visibility="advanced">
+        <Documentation>
+          Determine the batch size, proportion of the ensemble size (between 0 and 1).
+        </Documentation>
+        <DoubleRangeDomain name="range" min="0" max="1" />
+      </DoubleVectorProperty>
+      
+      <IntVectorProperty
+      name="Optimizer"
+      label="Optimizer"
+      command="SetOptimizer"
+      number_of_elements="1"
+      default_values="0"
+      panel_visibility="advanced">
+      <EnumerationDomain name="enum">
+          <Entry value="0" text="Adam"/>
+          <Entry value="1" text="SGD"/>
+          <Entry value="2" text="RMSprop"/>
+      </EnumerationDomain>
+        <Documentation>
+          Choose the optimizer method.
+        </Documentation>
+      </IntVectorProperty>
+      
+      <DoubleVectorProperty
+      name="GradientStepSize"
+      command="SetGradientStepSize"
+      label="Gradient Step Size"
+      number_of_elements="1"
+      default_values="0.1">
+        <Documentation>
+          Choose the step use for gradient descent.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="Beta1"
+      command="SetBeta1"
+      label="Beta1"
+      number_of_elements="1"
+      default_values="0.9"
+      panel_visibility="advanced">
+        <Documentation>
+          Coefficients used for computing running averages of gradient and its square.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="Beta2"
+      command="SetBeta2"
+      label="Beta2"
+      number_of_elements="1"
+      default_values="0.999"
+      panel_visibility="advanced">
+        <Documentation>
+          Coefficients used for computing running averages of gradient and its square.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="ReconstructionLossWeight"
+      command="SetReconstructionLossWeight"
+      label="ReconstructionLossWeight"
+      number_of_elements="1"
+      default_values="1.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the reconstruction loss in the optimization.
+        </Documentation>
+      </DoubleVectorProperty>
+      
+      <DoubleVectorProperty
+      name="TrackingLossWeight"
+      command="SetTrackingLossWeight"
+      label="TrackingLossWeight"
+      number_of_elements="1"
+      default_values="0.0"
+      panel_visibility="advanced">
+        <Documentation>
+          Weight of the tracking loss in the optimization.
```

Comment:
to discuss.

Reply:
thanks for adding a little documentation about the actual usage of the loss (low weight to trigger output origin initialization based on a copy from the input, recommended).

Reply:
Done in the next commit!